src/main/java/atdd/station/service/PathService.java

```diff
@@ -0,0 +1,42 @@
+package atdd.station.service;
+
+import atdd.station.model.Graph;
+import atdd.station.model.entity.Station;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class PathService {
+    @Autowired
+    private EdgeService edgeService;
+
+    @Autowired
+    private StationService stationService;
+
+    public List<Station> findPath(long startId, long endId) {
+        Graph graph = new Graph(edgeService.findAll());
+
+        List<Long> pathStationIds = graph.shortestDistancePath(startId, endId);
+
+        List<Station> stations = pathStationIds.stream()
+                .map(it -> stationService.findById(it))
+                .collect(Collectors.toList());
+
+        return stations;
+    }
+
+    public List<Station> findShortTimePath(long startId, long endId) {
+        Graph graph = new Graph(edgeService.findAll());
+
+        List<Long> pathStationIds = graph.shortestTimePath(startId, endId);
+
+        List<Station> stations = pathStationIds.stream()
+                .map(it -> stationService.findById(it))
+                .collect(Collectors.toList());
+
+        return stations;
```

Comment:
매번 findById를 하는 것도 좋지만 한번에 처리하는 방법도 있습니다 :)

```suggestion
        return stationService.findAllById(pathStationIds);
```